src/solver/variable/state.cpp

```diff
@@ -55,6 +55,9 @@ void ThermalState::StateForAnArea::initializeFromArea(const Data::Area& area)
     numberOfUnitsONbyCluster.resize(count);
     thermalClustersOperatingCost.resize(count);
     PMinOfClusters.resize(count);
```

Comment:
**PMinOfClusters** : we should make it a simple **double** local variable

Reply:
No, it's used in `solver/variable/economy/productionByDispatchablePlant.h:352`